plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -23,30 +23,26 @@ const (
 func TestWrite(t *testing.T) {
 	// Generate test metric with default name to test Write logic
 	floatMetric := testutil.TestMetric(1.0, "Float")
+	floatMetric.AddTag("host", "Host01")
+	floatMetric.AddTag("group", "Group01")
```

Comment:
Could you add a test case where the new tag isn't set to make sure the new change doesn't affect backward compatibility?

Reply:
done: 84458edb

Reply:
Thanks!